roles/sno_installer/vars/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
 # vars file for sno_installer
-cache_dir: "/opt/cache"
-pullsecret_file: "{{ cache_dir }}/pull-secret.txt"
+pullsecret_file: "{{ si_cache_dir }}/pull-secret.txt"
 default_libvirt_pool_dir: "/var/lib/libvirt/images"
 force_mirroring: false
 ocp_binary_path: "/usr/local/bin"
+
+snp_cache_dir: "{{ si_cache_dir }}"
```

Comment:
Is this variable correctly defined? Because the role is `sno_installer`, but prefix is `snp` (`sno_node_prep`?), so looks like used in a different role

Reply:
Yes, the name was intended.

This is something I noticed when adding the "si" prefix to the variables I'm adding or modifying.

I was wondering why some variables were defined in the "vars" directory instead of the "defaults" and the problem is that some variables defined in the sno_installer role are used by sno_node_prep, but for the later to have access to the variables the former must define them in the "vars" directory.

So "si_cache_dir" is a default for the sno_installer role, and "snp_cache_dir" is an input variable for the sno_node_prep role.

Now that I have a working version of the PR, I'll see to simplify and optimize it, so it may be I'll remove the variable after all.